app/src/main/java/com/android/unio/ui/event/EventCreation.kt

```diff
@@ -306,8 +305,14 @@ fun EventCreationScreen(
                         endDate = endTimestamp!!,
                         location = selectedLocation!!,
                         eventPictures = MockReferenceList(),
-                    ),
-                    onSuccess = { navigationAction.goBack() },
+                    )
+                eventViewModel.addEvent(
+                    inputStream,
+                    newEvent,
+                    onSuccess = {
+                      associationViewModel.addEventLocally(newEvent)
+                      navigationAction.goBack()
```

Comment:
Nice again, makes the app appear "refreshed" without requesting data it already has